pylint_django/transforms/foreignkey.py

```diff
@@ -108,10 +108,10 @@ def infer_key_classes(node, context=None):
                     # 'auth.models', 'User' which works nicely with the `endswith()`
                     # comparison below
                     module_name += '.models'
-                except ImproperlyConfigured:
+                except ImproperlyConfigured as exep:
                     raise RuntimeError("DJANGO_SETTINGS_MODULE required for resolving ForeignKey "
                                        "string references, see Usage section in README at "
-                                       "https://pypi.org/project/pylint-django/!")
+                                       "https://pypi.org/project/pylint-django/!") from exep
```

Comment:
I learned something new today, thanks! I don't even see this documented in the Python docs.

Reply:
I also only just learned about it, because pylint raised a warning in our CI linting :-)